### PR TITLE
suppress reporting, the default config of "enable all" was making

### PR DESCRIPTION
**Pull request by cartazio**
fine tuning hang!

also place the default models list in a location where it'll be easier to keep up to date

@ajn2004  this fixes the reporting issue i found wrt WandB prompt mid execution, and it also moves the config map into a location where it can be kept up to date with respect to supported models

@dorkitude  any style guidance or opinions welcome

### CODE REVIEW

**Comment by cartazio** [comment]
i should probably change the set construction to like frozen set or something?

File: src/llm_vm/config.py [hunk 1]
```diff
@@ -47,14 +47,7 @@
...
 # making MODELS_AVAILABLE a set because it will be used for membership testing
-MODELS_AVAILABLE = set([
-    "opt",
-    "bloom",
-    "neo",
-    "llama",
-    "gpt",
-    "chat_gpt",
-])
+MODELS_AVAILABLE = set(model_keys_registered)
```
Comment (cartazio) [inline src/llm_vm/config.py:50]: should this be a frozen set?

File: src/llm_vm/onsite_llm.py [hunk 2]
```diff
@@ -433,3 +434,21 @@ def finetune(self, dataset, optimizer, c_id):
...
+        "bloom": Small_Local_Bloom,
+        "neo": Small_Local_Neo,
+        "llama": Small_Local_LLama,
+        "pythia": Small_Local_Pythia,
+        "gpt": GPT3,
+        "chat_gpt": Chat_GPT,
+        "flan" : Small_Local_Flan_T5,
+        "pythia" : Small_Local_Pythia,
+        }
+model_keys_registered = __private_key_value_models_map.keys()
```
Comment (cartazio) [inline src/llm_vm/onsite_llm.py:450]: or i could make this the frozen set

**Comment by cartazio** [comment]
i'll merge this shortly if theres no highlighted problems

**Comment by ajn2004** [comment]
This looks like it's running on my end.

**Pull request merged**